lib/backend/k8s/conversion_test.go

```diff
@@ -846,6 +846,421 @@ var _ = Describe("Test NetworkPolicy conversion", func() {
 	})
 })
 
+// This suite of tests is useful for ensuring we continue to support kubernetes apiserver
+// versions <= 1.7.x, and can be removed when that is no longer required.
+var _ = Describe("Test NetworkPolicy conversion (k8s <= 1.7, no policyTypes)", func() {
```

Comment:
This is a copy of the NetworkPolicy tests from above, but each with the `policyTypes` section removed to make it look like it would appear on a k8s 1.7 cluster.

The assertions made in each test are the same.